0015/README.md

```diff
@@ -0,0 +1,153 @@
+---
+id: 0015
+title: HTTP Routing For Services
+status: ideation
+authors: Manny <manny@packet.com>
+---
+
+## Summary
+
+HTTP routing for services started by tink-worker running on "provisioner" machines.
```

Comment:
As a potential reviewer, I need more context here so my comments will be a bit meta and vague.

How does this relate to #18, is #18 a prerequisite? If so, what features of #18 are needed. How will this proposal change that behavior?